vizro-core/tests/unit/vizro/actions/_action_loop/test_get_action_loop_components.py

```diff
@@ -118,15 +105,7 @@ def managers_one_page_two_components_two_controls(vizro_app, dash_data_table_wit
 @pytest.fixture
 def managers_one_page_no_actions(vizro_app):
     """Instantiates managers with one "empty" page."""
-    vm.Dashboard(
-        pages=[
-            vm.Page(
-                id="test_page_no_actions",
-                title="Second page",
-                components=[vm.Card(text="")],
-            )
-        ]
-    )
+    vm.Dashboard(pages=[vm.Page(id="test_page_no_actions", title="Second page", components=[vm.Card(text="")])])
```

Comment:
This also looks more readable to me in the original format.

In general, I like condensing code in tests more than in src code, because the tests are more likely to be extended (by adding new tests) than to be changed or need to be understood in detail. However, this fixture is probably something that someone will have to understand.